Solutions/CohesitySecurity/Playbooks/Close-SentinelIncident-from-SNOW/Close-SentinelIncident-from-SNOW.json

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
Remove this playbook. We have marked this item for future work.

Reply:
removed.